static/js/components/ProfileImageUploader.js

```diff
@@ -10,18 +10,22 @@ import type { ImageUploadState } from '../reducers/image_upload';
 
 const onDrop = R.curry((startPhotoEdit, files) => startPhotoEdit(...files));
 
-const dropZone = onDrop => (
+const dropZone = (startPhotoEdit, setPhotoError) => (
   <Dropzone
-    onDrop={onDrop}
+    onDrop={onDrop(startPhotoEdit)}
     className="photo-active-item photo-dropzone"
     activeClassName="photo-active-item photo-dropzone active"
+    accept="image/*"
```

Comment:
indeed! I like it when people use standards. it accepts any valid MIME type, so you can do whatever you want!